web/controllers/user_controller.ex

```diff
@@ -68,4 +68,15 @@ defmodule CoursePlanner.UserController do
         |> redirect(to: user_path(conn, :index))
     end
   end
+
+  def notify(conn, _params) do
```

Comment:
Hm, it falls in the process ownership problem with the sandbox. I'll see what I can do.